### ISSUE TITLE
Add support for dark theme

### ISSUE THREAD

**Issue by kittykat** [matrix-org/conference-bot#46]
All the widgets should offer a dark theme alternative to fit in better with the user's theme and be more accessible for users who cannot use a light theme
